buf.yaml

```diff
@@ -0,0 +1,10 @@
+build:
+  roots:
+    - proto/internal
+    - proto/temporal-proto
+lint:
+  use:
+    - DEFAULT
+  except:
+    - PACKAGE_SAME_JAVA_MULTIPLE_FILES
+    - PACKAGE_SAME_JAVA_PACKAGE
```

Comment:
I added these two exceptions here instead of adding java options to server proto files.

Reply:
```
 $ buf check lint | wc -l                                    
0
```